packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -108,14 +109,22 @@ export class SDKClient {
         this.operatorAccountId = clientMain.operatorAccountId ? clientMain.operatorAccountId.toString() : 'UNKNOWN';
 
         // clear and create metrics in registry
-        const metricHistogramName = 'rpc_relay_consensusnode_response';
-        register.removeSingleMetric(metricHistogramName);
-        this.consensusNodeClientHistorgram = new Histogram({
-            name: metricHistogramName,
+        const metricHistogramCost = 'rpc_relay_consensusnode_cost';
```

Comment:
On principle I agree that the current name "rpc_relay_consensusnode_response" is not descriptive enough, sounds more like if we were capturing the time of the response as opposed as the costs of the call, however, this name is already being used on the dashboard and if we were to change it, there would be a `before` and `after` on the data on prometheus + grafana, so we would not be able to see anything before this change. and also we would need to change all the widgets that currently are using the old metric name and update them to use the new metric name.

So am not sure if is worth it to change the name, or perphas we can leave it as is, and just enrich the description of the metric.

Reply:
Yeah, description captures that it's cost so let's leave the name as is